src/main/java/software/amazon/qldb/QldbDriverBuilder.java

```diff
@@ -168,4 +169,6 @@
      * @return This builder object.
      */
     QldbDriverBuilder readAhead(int readAhead, ExecutorService executorService);
+
```

Comment:
I think we should take this chance to improve the docstrings. For one, this new builder option does not have a docstring.

On the SessionClientBuilder we should mention that the http client is overwritten, and a customized http wants to be used it needs to be passed into the new method, as well as briefly mentioning that the amount of connections should be equal or greater than the number of max concurrent transactions.

Reply:
Will update the docstrings later.